compiler/noirc_frontend/src/hir/def_map/mod.rs

```diff
@@ -243,6 +255,7 @@ pub struct Contract {
     pub name: String,
     pub location: Location,
     pub functions: Vec<ContractFunctionMeta>,
+    pub events: Vec<StructId>,
```

Comment:
can you modify the doc comment for Contract above to specify that it also includes events